exsim/ltem.py

```diff
@@ -0,0 +1,92 @@
+import numpy as np
+import discretisedfield as df
+import micromagneticmodel as mm
+from scipy import constants
+
+
+# TODO
+# - remove computation of wavelength (not required)
+# - recheck that all units are correct, everything should be in SI units
+# - if possible: remove transposing before/after Fourier transform
+# - Write a proper documentation.
+# - Cs is not used. Can it be removed?
+# - Write tests.
+def ltem_phase(field, /, U, Cs, kx=0.1, ky=0.1):
+    """LTEM phase contrast.
+
+    Parameters
+    ----------
+    field : df.Field
+        Magnetisation field.
+    U : numbers.Real
+        Accelerating voltage of electrons in V.
+    Cs : numbers.Real
+        Spherical aberration coefficient
+    kx : numbers.Real, optional
+        Tikhonov filter radius in x in pixels.
+    ky : numbers.Real, optional
+        Tikhonov filter radius in y in pixels.
+
+    Returns
+    -------
+    df.Field
+        Phase in real space.
+    df.Field
+        Phase in Fourier space.
+
+    Example
+    -------
+    ...
+    """
+    const = 1j * field.mesh.region.edges[2] / (
+        2 * constants.codata.value('mag. flux quantum'))
+    m_sat = np.max(field.norm.array) * mm.consts.mu0
+
+    wavelength = _relativistic_wavelength(U)
```

Comment:
If there are similar "helper" functions in other files, we might make `util` submodule.

Reply:
Totally agree. I also thought about moving it to a `util` submodule in case there are similar functions or we need it in different places.

Reply:
For better clarity. Did you think of
(a)
```
exsim
|---__init__.py
|---ltem.py
|---util.py
```
or
(b)
```
exsim
|---__init__.py
|---ltem.py
|---util
|   |---__init__.py
|   |---util.py
```

Reply:
I was thinking about option (b) simply because this is the structure we have in other packages.

Reply:
As discussed in https://github.com/ubermag/exsim/pull/7#discussion_r649790679 calculation of the wavelength is not necessarily a helper function and not required at this position. We might therefore think about not moving it to `utils` but instead make it public. Depends on where else in the code it is used.